seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -893,50 +893,50 @@
       "hints": [],
       "type": "waypoint",
       "challengeType": 1,
-      "translations": {}
+      "translations": {}      
```

Comment:
Remove white space, refer out contributing guide to configure the linter for your text editor.